src/main/sensors/acceleration.c

```diff
@@ -57,14 +57,9 @@ void accUpdate(timeUs_t currentTimeUs, rollAndPitchTrims_t *rollAndPitchTrims)
     acc.isAccelUpdatedAtLeastOnce = true;
 
     for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
-        DEBUG_SET(DEBUG_ACCELEROMETER, axis, acc.dev.ADCRaw[axis]);
-        acc.accADC[axis] = acc.dev.ADCRaw[axis];
-    }
-
-    if (accelerationRuntime.accLpfCutHz) {
-        for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
-            acc.accADC[axis] = biquadFilterApply(&accelerationRuntime.accFilter[axis], acc.accADC[axis]);
-        }
+        const int16_t val =  acc.dev.ADCRaw[axis];
+        DEBUG_SET(DEBUG_ACCELEROMETER, axis, val);
+        acc.accADC[axis] = accelerationRuntime.accLpfCutHz ? pt2FilterApply(&accelerationRuntime.accFilter[axis], val) : val;
```

Comment:
It would be great to have some decent simulator, so that results of filters are compared by flight performance. Better looking trace on time-domain graph may be quite misleading.

Reply:
Agree. I have introduced 3 new filters so we need to decide 3 filter types and 3 cutoffs. At least.

Reply:
@ledvinap I'm on it (the simulator part)